stable/kube-lego/values.yaml

```diff
@@ -34,6 +34,9 @@ podAnnotations: {}
 
 replicaCount: 1
 
+## serviceAccount is the name of the ServiceAccount to use to run the kube-lego Pod.
+#serviceAccount: 
```

Comment:
Create a top-level object named `rbac` and move this value to `rbac.serviceAccountName`. Set the default value to `default`. Also add an `rbac.create` value and set to `false`.